optout/src/main/resources/application.optout.properties

```diff
@@ -0,0 +1,12 @@
+#------------------------------------------------------------------------------------ QUARTZ SCHEDULER CONFIG
+spring.quartz.job-store-type=jdbc
+spring.quartz.jdbc.initialize-schema=always
+spring.quartz.properties.org.quartz.jobStore.isClustered=true
+spring.quartz.properties.org.quartz.scheduler.instanceId=AUTO
+spring.quartz.properties.org.quartz.jobStore.tablePrefix=qrtz_
+spring.quartz.properties.org.quartz.jobStore.driverDelegateClass=org.quartz.impl.jdbcjobstore.PostgreSQLDelegate
+
```

Comment:
The S3 properties need to be added here and refer to corresponding environment variables.

Reply:
In my local, it read directly from my environment variable into the property inside `OptOutProcessorImpl` using the `@Value` annotation